### PR TITLE
feat: pergel

### PR DESCRIPTION
**Pull request by productdevbook**
Hi Team, 

Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project kickstarts and seamless integration of various modules.

### CODE REVIEW

File: modules/pergel.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: pergel
+description: >-
+  Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
+  kickstarts and seamless integration of various modules.
+repo: oku-ui/pergel
+npm: pergel-system
```
Comment (productdevbook) [inline modules/pergel.yml:6]: '@pergel/nuxt'

how to change, autofix change :/

Comment (danielroe) [reply modules/pergel.yml:6]: It's picking it up from your github repo

File: modules/pergel.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+name: pergel
+description: >-
+  Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
+  kickstarts and seamless integration of various modules.
+repo: oku-ui/pergel
```
Comment (danielroe) [inline modules/pergel.yml:5]:
```suggestion
repo: oku-ui/pergel#main/packages/nuxt
```

File: modules/pergel.yml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+name: pergel
+description: >-
+  Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
+  kickstarts and seamless integration of various modules.
+repo: oku-ui/pergel#main/packages/nuxt
+npm: '@pergel/nuxt'
```
Comment (productdevbook) [inline modules/pergel.yml:6]:
```suggestion
npm: 'pergel'
```

File: modules/pergel.yml [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+github: https://github.com/oku-ui/pergel
+website: https://oku-ui.com/pergel
+learn_more: ''
+category: Devtools
+type: 3rd-party
+maintainers:
+  - name: Mehmet - productdevbook
+    github: productdevbook
+compatibility:
+  nuxt: ^3.9.0
```
Comment (productdevbook) [inline modules/pergel.yml:17]:
```suggestion
  nuxt: ^3.10.0
```

File: modules/pergel.yml [hunk 5]
```diff
@@ -3,7 +3,7 @@ description: >-
   Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
   kickstarts and seamless integration of various modules.
 repo: oku-ui/pergel#main/packages/nuxt
-npm: 'pergel'
+npm: '@pergel/nuxt'
```
Comment (productdevbook) [inline modules/pergel.yml:6]: nuxt please change -> '@pergel/nuxt' -> 'pergel'

Comment (danielroe) [reply modules/pergel.yml:6]: But the Nuxt module is `@pergel/nuxt`, right?

https://github.com/oku-ui/pergel/blob/main/packages/nuxt/package.json#L2

Comment (productdevbook) [reply modules/pergel.yml:6]: 'pergel/nuxt' all modules are gathered into a single structure, yes, that's true, but we don't use it.

`pnpm i pergel` after add nuxt.config.ts 'pergel/nuxt'

File: modules/pergel.yml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+name: pergel
+description: >-
+  Pergel is a tailor-made solution for Nuxt and Nitro, providing swift project
+  kickstarts and seamless integration of various modules.
+repo: oku-ui/pergel#main/packages/nuxt
```
Comment (danielroe) [inline modules/pergel.yml:5]:
```suggestion
repo: oku-ui/pergel#main/packages-core/pergel
```

**Comment by productdevbook** [comment]
i approve ❤️

**Review by danielroe — APPROVED** [review]

**Pull request merged**